zmdocument.cls

```diff
@@ -7,6 +7,7 @@
 
 \RequirePackage[utf8]{inputenc}
 \RequirePackage[T1]{fontenc}
```

Comment:
Useless with LuaLaTeX.

Reply:
This is object of next PR :)

Reply:
Yeah, ok, but I agree with @Karnaj here :)